witchcraft/internal/middleware/route.go

```diff
@@ -107,3 +115,37 @@ func NewRouteLogTraceSpan() wrouter.RouteHandlerMiddleware {
 		next(rw, req, reqVals)
 	}
 }
+
+// NewRoutePanicRecovery returns a middleware which recovers panics within the inner route handler.
+// This is distinct from NewRequestPanicRecovery in that it runs when all logging/telemetry are configured on the request.
+func NewRoutePanicRecovery() wrouter.RouteHandlerMiddleware {
```

Comment:
Is there any way to add a test for this?

Reply:
This is tested in the integration test by the routes that panic inside the handler functions

Reply:
Ah, I see - got it.